tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+- name: Install MySQL
+  apt:
+    name={{ item }}
+    state=present
+    update_cache=true
+    cache_valid_time=3600
+  with_items:
+  - mysql-server
+
+- name: Generate MySQL config
+  template:
+    src=templates/mysql.cnf.j2
+    dest=/etc/mysql/conf.d/mysql.cnf
+  register: updated_config
+
+- name: (re)Start MySQL
+  service:
+    name=mysql
+    enabled=yes
+    state={{ 'restarted' if updated_config.changed else 'running' }}
+
+- name: Create MySQL user
+  shell: >
+    # Check for the user
+    (echo "SELECT user, host FROM mysql.user;" | mysql | grep {{ mysql_database_user }})
+    ||
+    # If not present, create
+    (echo "CREATE USER '{{ mysql_database_user }}'@'localhost' IDENTIFIED BY '{{ mysql_database_user_password }}';" | mysql)
+
+- name: Generate new database script
+  template:
+    src=templates/create_new_database.sh.j2
+    dest={{ mysql_new_database_script }}
+    mode=777
+
+- name: Create new MySQL database
+  shell: sh {{ mysql_new_database_script }}
```

Comment:
Yes! Except it didn't because the script doesn't have `set -e`, so I'll add that and also have a check for the db existing @_@